docs/programming-guide.md

```diff
@@ -24,7 +24,8 @@ along with if you launch Spark's interactive shell -- either `bin/spark-shell` f
 
 <div data-lang="scala"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} uses Scala {{site.SCALA_BINARY_VERSION}}. To write
+Spark {{site.SPARK_VERSION}} is built and distributed to work with Scala {{site.SCALA_BINARY_VERSION}} 
+by default. (Spark can be built to work with other versions of Scala, too.) To write
```

Comment:
Ahhh, that's super helpful and would be good to know that what's built in Maven is different than what you use to run a cluster. Is this a client-server setup? Like can you use the 2.11 artifacts from Maven in a Spark client and run on a 2.10 cluster?

Maybe it'd be better to say "The Spark {{site.SPARK_VERSION}} cluster is built and distributed". My employer maintains the cluster. I just deal with artifacts from Maven. So in my eyes everything was supported equally in both versions since I could find both versions in Maven, but now I see that I'm only seeing half the puzzle. I don't have a good understanding from this portion of how they fit together.